apostrophecms/a3-demo
# adds widget previews to main

## Summary

Updates the `main` area of the demo to use the new widget preview feature.

## What are the specific steps to test this change?

1. Start the demo
1. Login and try to edit the `main` area on the homepage.
1. The expanded widget preview menu should open with images for the container widget and the gh pull request widget.

## What kind of change does this PR introduce?


## Make sure the PR fulfills these requirements:
